package.json

```diff
@@ -85,7 +85,7 @@
     ]
   },
   "dependencies": {
-   "@scalescss/scalescss": "6.0.0-alpha.976c5031",
+    "@scalescss/scalescss": "6.0.0-alpha.976c5031",
```

Comment:
the scales import is something that currently isnt working, but added a missing space here anyway